### PR TITLE
Add Link to Manage Protocol Page Addresses #80

### PR DESCRIPTION
**Pull request by AliSchlereth**
@h-m-m 
Add link to New Protocol from Manage Protocol page. 
Put Protocols in <ul> tags and added an hr for more clear distinction between the links.
Add a basic feature test.

Let me know if there are further navigation needs or updates you'd like to see.
Addresses #80

### CODE REVIEW

**Comment by h-m-m** [comment]
Hi, Ali. Thanks for adding this. I especially appreciate the addition of a test in this PR.

File: app/views/survey_protocols/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,10 @@
+<h2>Protocols</h2>
 <% @survey_protocols.each do |protocol| %>
-    <%= link_to protocol.title, [:edit, protocol] %>
+    <ul><%= link_to protocol.title, [:edit, protocol] %></ul>
```
Comment (h-m-m) [inline app/views/survey_protocols/index.html.erb]: @AliSchlereth: Should this have more of a `<ul><li></li></ul>` structure?

File: app/views/survey_protocols/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,10 @@
-<% @survey_protocols.each do |protocol| %>
-    <%= link_to protocol.title, [:edit, protocol] %>
-    <br/>
-<% end  %>
+<h2>Protocols</h2>
+<ul>
+  <% @survey_protocols.each do |protocol| %>
+      <li><%= link_to protocol.title, [:edit, protocol] %></li>
+  <% end  %>
+</ul>
```
Comment (AliSchlereth) [inline app/views/survey_protocols/index.html.erb:6]: How about this?

Comment (h-m-m) [reply app/views/survey_protocols/index.html.erb:6]: Great!

**Pull request merged**